org.eclipse.winery.repository.rest/src/test/resources/entitytemplates/ShipOrderTemplate_Properties.xml

```diff
@@ -1,5 +1,6 @@
 <Properties xmlns="http://docs.oasis-open.org/tosca/ns/2011/12">
-	<w3c:shiporder xmlns:artifacttypes="http://winery.opentosca.org/test/artifacttypes" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:tosca="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:w3c="https://www.w3schools.com/xml/schema_example.asp" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" orderid="123456">
+	<w3c:shiporder xmlns:artifacttypes="http://winery.opentosca.org/test/artifacttypes" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:tosca="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:w3c="https://www.w3schools.com/xml/schema_example.asp" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/
```

Comment:
This is wrong -> the namespace is split on two lines. I think, the left version is correct and the proposed version should not be used.